### PR TITLE
Course card

### PR DESCRIPTION
**Pull request by MagicianJoshua**
Made some significant progress on the user dashboard and the browse courses area

### CODE REVIEW

File: client/src/components/mainComponents/CoursesContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import coursesInfo from "../../utils/courseTest";
+import CourseCard from "../miniComponents/CourseCards";
+import "../../assets/css/CourseContainer.css";
+
+export default function Courses() {
```
Comment (bhansi) [inline client/src/components/mainComponents/CoursesContainer.jsx]: Please change the function name to CoursesContainer in order to match the file name.

File: client/src/components/miniComponents/CourseCards.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import "../../assets/css/courseCard.css";
+import placeholder from "../../assets/img/CookingCourse.png";
+
+export default function CourseCard(props) {
```
Comment (bhansi) [inline client/src/components/miniComponents/CourseCards.jsx]: Please either change the file name to CourseCard or the function name to CourseCards.

File: client/src/components/miniComponents/ProfileSideMenu.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { Button } from "bootstrap";
+import "../../assets/css/ProfileSideMenu.css";
+import { useState } from "react";
+
+export default function SideMenu(props) {
```
Comment (bhansi) [inline client/src/components/miniComponents/ProfileSideMenu.jsx]: Please either change the file name to SideMenu or the function name to ProfileSideMenu.

File: client/src/components/miniComponents/SettingsPage.jsx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+export default function Settings(props) {
```
Comment (bhansi) [inline client/src/components/miniComponents/SettingsPage.jsx]: Please change the file name to Settings or the function name to SettingsPage.

**Review by bhansi — CHANGES_REQUESTED** [review]

File: client/src/components/mainComponents/CoursesContainer.jsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import coursesInfo from "../../utils/courseTest";
+import CourseCards from "../miniComponents/CourseCards";
+import "../../assets/css/CourseContainer.css";
+
+export default function CourseContainer() {
```
Comment (bhansi) [inline client/src/components/mainComponents/CoursesContainer.jsx]: Please make the file name and function name match.

**Review by bhansi — DISMISSED** [review]

**Review by Valleyleaf — APPROVED** [review]
Approved.

**Pull request merged**